cli/tests/node_compat/test.ts

```diff
@@ -83,6 +83,7 @@ async function runTest(t: Deno.TestContext, path: string): Promise<void> {
         "--quiet",
         "--unstable",
         //"--unsafely-ignore-certificate-errors",
+        "--unstable-bare-node-builtins",
```

Comment:
Why is this flag required?

Reply:
The added case (`test-parse-args.mjs`) is the first one in our compat test suite that depends on  `cli/tests/node_compat/test/common/index.mjs`, and that file has bare import of `module` module https://github.com/denoland/deno/blob/45fe3ccccfc2db9a969d2e6d258011ae96ee7d1e/cli/tests/node_compat/test/common/index.mjs#L8